.github/workflows/issue.yml

```diff
@@ -24,12 +24,30 @@ jobs:
         summary: « [hugr] ${{ github.event.issue.title }}»
         description: ${{ github.event.issue.html_url }}
         fields: '{"labels": ["hugr"]}'
+    - name: set_priority_var
```

Comment:
```suggestion
    - name: Parse the priority label into a Jira priority
      id: set_priority_var
```

Reply:
Nice, I didn't know you could use a seperate `id`